### ISSUE TITLE
Darknet maintained fork support

### ISSUE THREAD

**Issue by HagegeR** [lutzroeder/netron#381]
Hi,

I just found you'r repo and it's awesome!

I work mostly with darknet lately and I use [@AlexeyAB](https://github.com/AlexeyAB/darknet) fork. It has a lot of improvements over the original and, most important, it's still alive and has incorporated a lot of new types of layers!

I'd love to see your repo work on this Darknet,

thank you for your time :)

**Comment by HagegeR** [comment]
@AlexeyAB what do you think about this repo? I think this can also help put the documentation on layers directly in sight, exposing the possible parameters ect.

**Comment by AlexeyAB** [comment]
Yes, it would be nice if this repo will support new layers and models, like Gaussian-yolo, EfficientDet, ASFF-Yolo, ...:
* EfficientNetB0-PRN-Yolov3 https://github.com/AlexeyAB/darknet/blob/master/cfg/enet-coco.cfg  with [weights file enetb0-coco_final.weights](https://drive.google.com/file/d/1FlHeQjWEQVJt0ay1PVsiuuMzmtNyv36m/view)

* MixNet-M-GPU: 
[mixnet_m_gpu.cfg.txt](https://github.com/lutzroeder/netron/files/3935529/mixnet_m_gpu.cfg.txt)


* CSPResNext50-panet-spp-Yolo: https://raw.githubusercontent.com/WongKinYiu/CrossStagePartialNetworks/master/cfg/csresnext50-panet-spp.cfg with [weights file 
csresnext50-panet-spp_final.weights ](https://drive.google.com/open?id=1Y6vJQf-Vu9O0tB10IUYNttktA-DLp5T1) 

![cmp](https://user-images.githubusercontent.com/4096485/70380370-4ccd8380-194b-11ea-9429-4617d31df676.png)

**Comment by lutzroeder** [comment]
@HagegeR @AlexeyAB can you please create issues for specific examples and repros of what is not working and submit PRs with fixes.

**Comment by HagegeR** [comment]
Ok so I'm trying to list the changes that needs to be done:
currently partially supported layers/activation are: conv/deconv, dropout, max/avgpool, connected, softmax, gru, lstm, rnn, crop, reorg, shortcut, batch_norm, norm, route, relu, leaky, linear, logistic, ramp, upsample, and yolo.
so missing things are: 
- [ ] gaussian yolo, scale_channels, swish, mish, cost(?), and region
- [ ] the net part itself is not documented as well, options such as letterbox, policy, angle, hue, saturation, exposure, aspect mixup, and blur, are missing too
- [ ] iou_loss, iou_normalizer, cls_normalizer, scale_x_y, and are not present in the current yolo layers
- [ ] xnor in conv

did I miss some new features maybe?
another idea: can this repo also check for errors in cfg? like num of filters not coordinated with class num and such?

**Comment by HagegeR** [comment]
I have started to work on the metadata, I think It will take me 2-3 days [https://github.com/HagegeR/netron/pull/1](url)

**Comment by lutzroeder** [comment]
@HagegeR rebase to tip. HagegeR#1 doesn't have the latest `darknet-metadata.json` changes.

**Comment by HagegeR** [comment]
thanks for merging it, there is still some work to do such as:

- [ ] the net part lacks improvements made in the fork, options such as max_batches, burn_in, scales, steps, policy, letterbox, power, mixup, and blur, are missing

- [ ] the from attribute that is (apparently) some kind of shortcut, I didn't know how to implement it

- [ ] types from part of the attributes

- [ ] description for principal parameters in the format of help as onnx does

- [ ] looking at [https://github.com/AlexeyAB/darknet/blob/master/cfg/enet-coco.cfg](url) as an example, there seems to be orphaned yolo layers (not connected to any output) so probably I missed something

**Comment by HagegeR** [comment]
I've had a look at a lot of the supported framework. Can the yolo input/output format be copied from Core ML format? a lot of documentation there fits for most if not all of Darknet output, look at this for an example: [https://docs-assets.developer.apple.com/coreml/models/Image/ObjectDetection/YOLOv3Tiny/YOLOv3Tiny.mlmodel](url)

**Comment by HagegeR** [comment]
Thank you for your support in this endeavour!

> @lutzroeder Can the yolo input/output format be copied from Core ML format?

is there any way for this to happen?
